e3sm_to_cmip/mpas.py

```diff
@@ -226,9 +228,14 @@ def get_sea_floor_values(ds, dsMesh):
 
 
 def open_mfdataset(fileNames, variableList=None,
-                   chunks={'nCells': 32768, 'Time': 6}):
+                   chunks={'nCells': 32768, 'Time': 6}, daskThreads=6):
     '''Open a multi-file xarray Dataset, retaining only the listed variables'''
 
+    dask.config.set(schedular='threads',
+                    pool=ThreadPool(min(multiprocessing.cpu_count(),
+                                        daskThreads)))
```

Comment:
I'm capping each subprocess at 6 dask threads.  This could be made fewer if need be but it seems likely to be safe. The default is to use all cores, which isn't helpful with multiple processes.  @sterlingbaldwin, let me know how this goes.  Should speed things up because processes aren't competing for the same cores.

Reply:
I appreciate changing the default. I find it very annoying when a process decides it needs to grab all the cores at once by default. 6 seams reasonable, if I run into anything I'll hook it up to use the --nproc command line flag.